server_monitoring/README.rst

```diff
@@ -0,0 +1,87 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+server monitoring
+=================
+
+This module allows in-database logging of some statistics in order to monitor
+the health of an openerp instance.
+
+Database indicators are logged (number of rows, table size, number of reads,
+number of updates...), with a cron running each week by default. This cron
+needs to be activated after the module is installed.
+
+Some process indicators are logged (cpu time, memory) together with information
+about the different XMLRPC calls made to the server (user, model, method).
+
+Two crons are provided to cleanup old logs from the database.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Settings -> Scheduled Actions and tune the cron tasks
```

Comment:
What about this?
#. `Settings -> Scheduled Actions` and tune the cron tasks
#. `Settings -> Configuration -> General Settings`
#. Enable Monitor RPC Calls if you want information about RPC calls 
being made (causes some performance overhead).